analyzers/nodejs/nodejs.go

```diff
@@ -261,6 +263,18 @@ func (a *Analyzer) Analyze() (graph.Deps, error) {
 	return yarn.FromProject(filepath.Join(a.Module.BuildTarget, "package.json"), filepath.Join(a.Module.BuildTarget, "yarn.lock"))
 }
 
+// sanitizeBuildTarget ensures that legacy behavior stays intact but users are warned if it is implemented.
+func (a *Analyzer) sanitizeBuildTarget() {
+	if strings.HasSuffix(a.Module.BuildTarget, "/package.json") {
+		log.Warn("Specifiying the package.json file as a module's target in fossa's config file is no longer supported. Only the directory is neccessary.")
```

Comment:
Instead of the second sentence, I'd prefer: `Instead, the target should specify the path to the project folder.`